app/assets/javascripts/controllers/ems_keypair/ems_keypair_controller.js

```diff
@@ -63,7 +63,9 @@
   };
 
   EmsKeypairController.prototype.showValidate = function(tab) {
-    return !(this.model.emstype == 'openstack_infra' && this.newRecord && tab == 'ssh_keypair')
+    openstack_and_new = this.model.emstype == 'openstack_infra' && this.newRecord
+    rhevm = this.model.emstype == 'rhevm'
+    return !((openstack_and_new || rhevm) && tab == 'ssh_keypair')
   };
```

Comment:
A few good coding practices in javascript are recommended here -
1. define variables
2. use camelCase
3. end the line with semicolon
4. use `===`
5. whitespace after `!`

```js
var newOpenstack = this.model.emstype === 'openstack_infra' && this.newRecord;
var rhevm = this.model.emstype === 'rhevm';
return ! ((newOpenstack || rhevm) && tab === 'ssh_keypair');
```